### PR TITLE
Raidboss: A1S minor trigger fixes

### LINKED ISSUE #1314: A1S triggers no calling
I noticed this problem, the triggers (gunnery pod) and (resin bomb) are not doing their job.
 a problem has identified (ID on resin bomb), but it is not enough.
I tried a lot of things, but I can't find it, I tried with the French and English version.
I'm sorry to bother you for old content. Thank you.

Here are screens and the log file:
- https://uptobox.com/jj5us74d2ess act file a1s
- https://uptobox.com/m7m1g7efnfxf htlm file a1s

no calling resin bomb (infotext)
resin

no caling gunnery pod (respons.js)
gunnery

### PR DESCRIPTION
**Pull request by JLGarber**
This should resolve #1314. Unfortunately it doesn't look like the log currently has targets available for Resin Bomb, so we just have to fall back on warning everyone.

(`E46` is the castbar when Oppressor is targeting itself, then `E47` is the puddles on the floor ready to go off.)

### CODE REVIEW

**Comment by VanSlanzar** [comment]
I just did (a1s). The problem for (gunnery pod) is solved. However, (resin bomb) still not. Either ID (e46 or e47), I don't know, and could there be a problem with the file (a1s.txt)? Thank you for help me with this problem.

**Comment by JLGarber** [comment]
No, being alone would make no difference. Oppressor is not targeting specific players at any time during the Resin Bomb mechanic. (Or, at least, is not targeting them through currently-available log lines.)

Unfortunately, I do not know for sure what to tell you. It Works On My Machine. From everything I can see in the screenshot you posted, you should have received an info popup when Oppressor started the first cast. Is it possible you have that trigger disabled somewhere?

**Comment by VanSlanzar** [comment]
How is possible ?

**Comment by quisquous** [comment]
In the log file, what do the `14:` log files look like for resin bomb?

**Comment by JLGarber** [comment]
```
[13:08:10.555] 14:E46:Oppressor starts using Resin Bomb on Oppressor.
[13:08:14.541] 15:40000571:Oppressor:E46:Resin Bomb:40000571:Oppressor:3C:0:1B:[etc]

[13:08:15.514] 14:E47:Oppressor starts using Resin Bomb on Unknown.
[13:08:18.481] 15:40000572:Oppressor:E47:Resin Bomb:E0000000:[etc]
```

(Timestamps are included for timing comparison.)

I don't usually run with full packet data enabled, so I don't know whether there's anything useful in there, but I would assume that the snapshot for who has the invisible prey markers is at the end of the `E46` cast.

@VanSlanzar , would you be able to pull these four lines out of your log so we can compare them?

**Comment by VanSlanzar** [comment]
I'm not sure what you want me to do.

**Comment by VanSlanzar** [comment]
- [15:15:07.000] 00:282b:The oppressor readies Resin Bomb.
- [15:15:07.459] 14:E46:Oppressor starts using Resin Bomb on Oppressor.
- [15:15:11.433] 15:40009C4C:Oppressor:E46:Resin Bomb:40009C4C:Oppressor:3C:0:1B:E468000:0:0:0:0:0:0:0:0:0:0:0:0:959474:1411040:8840:10000:0:1000:-0.01531982:-170.7028:-23.92503:-3.130184:959474:1411040:8840:10000:0:1000:-0.01531982:-170.7028:-23.92503:-3.130184:0000675C
- [15:15:12.414] 14:E47:Oppressor starts using Resin Bomb on Unknown.
- [15:15:12.000] 00:282b:The oppressor uses Resin Bomb.
- [15:15:15.407] 15:40009C4D:Oppressor:E47:Resin Bomb:E0000000::0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:::::::::::160000:160000:8840:10000:0:1000:-0.05175314:-174.3965:-23.85911:-4.792213E-05:00006766
- [15:15:15.000] 00:302b:The oppressor uses Resin Bomb.
- [15:16:35.000] 00:282b:The oppressor readies Resin Bomb.
- [15:16:35.000] 00:282b:The oppressor 0.5 readies Resin Bomb.
- [15:16:36.061] 14:E46:Oppressor starts using Resin Bomb on Oppressor.
- [15:16:36.061] 14:E46:Oppressor 0.5 starts using Resin Bomb on Oppressor 0.5.
- [15:16:40.025] 15:40009C4C:Oppressor:E46:Resin Bomb:40009C4C:Oppressor:3C:0:1B:E468000:0:0:0:0:0:0:0:0:0:0:0:0:309589:1411040:8840:10000:0:1000:0.6866455:-171.1605:-23.90001:1.036268:309589:1411040:8840:10000:0:1000:0.6866455:-171.1605:-23.90001:1.036268:000067F2
- [15:16:40.025] 15:40009CAA:Oppressor 0.5:E46:Resin Bomb:40009CAA:Oppressor 0.5:3C:0:1B:E468000:0:0:0:0:0:0:0:0:0:0:0:0:331829:1411040:8840:10000:0:1000:1.785278:-172.1981:-23.8938:0.7510388:331829:1411040:8840:10000:0:1000:1.785278:-172.1981:-23.8938:0.7510388:000067F3
- [15:16:41.045] 14:E47:Oppressor starts using Resin Bomb on Unknown.
- [15:16:41.045] 14:E47:Oppressor 0.5 starts using Resin Bomb on Unknown.
- [15:16:41.000] 00:282b:The oppressor 0.5 uses Resin Bomb.
- [15:16:41.000] 00:282b:The oppressor uses Resin Bomb.
- [15:16:44.039] 15:40009C4D:Oppressor:E47:Resin Bomb:E0000000::0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:::::::::::160000:160000:8840:10000:0:1000:7.470429:-177.542:-23.91435:-4.792213E-05:000067FD
- [15:16:44.039] 15:40009CAB:Oppressor 0.5:E47:Resin Bomb:E0000000::0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:::::::::::160000:160000:8840:10000:0:1000:7.470429:-177.542:-23.91435:-4.792213E-05:00006801
- [15:16:44.000] 00:302b:The oppressor 0.5 uses Resin Bomb.
- [15:16:44.000] 00:302b:The oppressor uses Resin Bomb.

https://zupimages.net/viewer.php?id=20/15/nrrg.png

**Comment by JLGarber** [comment]
Yes, that is what we needed. (Unfortunately I cannot use the upload you provided as it was closed to users from the US.)

I have no idea what could be causing the problem for you, I just now went in again and it worked fine for me. Maybe you need to copy the updated file into your `raidboss` folder again?

**Comment by VanSlanzar** [comment]
I did a complete installation last night, everything is new.
I'm waiting for the next update to see.
I think I had done too much, and I may have corrupted my files or something.

**Comment by JLGarber** [comment]
Ahh, that makes more sense. That being the case, this is probably safe to merge prior to the next release.

File: ui/raidboss/data/03-hw/raid/a1s.js [hunk 1]
```diff
@@ -69,8 +69,7 @@
     },
     {
       id: 'A1S Resin Bomb',
-      regex: Regexes.startsUsing({ id: 'E47', source: 'Oppressor' }),
-      condition: Conditions.targetIsYou(),
+      regex: Regexes.startsUsing({ id: 'E46', source: 'Oppressor', capture: false }),
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a1s.js:72]: Should this be `['E46', 'E47']`? It looks like these are both resin bomb ids, or am I misunderstanding?

File: ui/raidboss/data/03-hw/raid/a1s.js [hunk 2]
```diff
@@ -17,15 +17,15 @@
         cn: '上升',
       },
     },
-  ],
-  triggers: [
     {
       id: 'A1S Gunnery Pod',
-      regex: Regexes.startsUsing({ id: 'E41', source: 'Oppressor', capture: false }),
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a1s.js:24]: Oh right, there's no startsUsing here, sorry!

File: ui/raidboss/data/03-hw/raid/a1s.js (hunk 1 above)
Comment (JLGarber) [reply ui/raidboss/data/03-hw/raid/a1s.js:72]: Resin Bomb is a two-step cast, it's not a multiple-actors-doing-the-same-thing situation. You can see it in this [timestamped video](https://youtu.be/-yYS25mR7CI?t=19). You will see a long cast first, at the end of which four players are marked up with puddles. This first cast is `E46` and is the one that is important to stack for. The puddles activating is an invisible `E47` cast.

**Comment by VanSlanzar** [comment]
Is there still a problem to correct?

**Comment by quisquous** [comment]
Good to go, thanks!

**Pull request merged**